### PR TITLE
Adds created_at timestamps to operations and payments

### PR DESCRIPTION
**Pull request by nullstyle**

### CODE REVIEW

File: src/github.com/stellar/horizon/actions_payment.go [hunk 1]
```diff
@@ -48,10 +53,17 @@ func (action *PaymentsIndexAction) SSE(stream sse.Stream) {
 			records := action.Records[stream.SentCount():]
 
 			for _, record := range records {
-				res, err := resource.NewOperation(action.Ctx, record)
+				ledger, found := action.Ledgers.Records[record.LedgerSequence()]
```
Comment (bartekn) [inline src/github.com/stellar/horizon/actions_payment.go:57]: Shouldn't it also call `action.loadLedgers` before this function?

**Review by bartekn — COMMENTED** [review]
LGTM! Adding tests would be great.

File: src/github.com/stellar/horizon/actions_payment.go (hunk 1 above)
Comment (nullstyle) [reply src/github.com/stellar/horizon/actions_payment.go:57]: Good catch!

**Comment by nullstyle** [comment]
Adding some tests now.

**Pull request merged**